checkout-core/src/main/java/com/adyen/checkout/core/api/Environment.java

```diff
@@ -46,6 +47,7 @@ public Environment createFromParcel(@NonNull Parcel in) {
     static {
         try {
             TEST = new Environment(new URL("https://checkoutshopper-test.adyen.com/checkoutshopper/"));
+            LIVE = new Environment(new URL("https://checkoutshopper-live.adyen.com/checkoutshopper/"));
```

Comment:
Maybe if you do `LIVE = EUROPE` it's a bit more clear that the default environment is the european one and the validation below would still work since they are the same object. What do you think?

Reply:
good idea